uxarray/grid.py

```diff
@@ -225,3 +227,34 @@ def __init_ds_var_names__(self):
             "nMesh2_face": "nMesh2_face",
             "nMaxMesh2_face_nodes": "nMaxMesh2_face_nodes"
         }
+
+    def __init_ds_var_methods__(self):
+        """Initialize variables for directly accessing Coordinate and
+        Data variables through ugrid conventions
+
+        Examples
+        ----------
+        Initialize some Grid Object
+        >>> grid = ux.open_dataset(file_path)
+
+        Return Xarray Data Array
+        >>> x = grid.Mesh2_node_x      
```

Comment:
What about providing this "Return Xarray data array " part as follows (it is a bit long though):

Assuming the mesh node coordinates for longitudes come from the input in the name "mesh_node_x". We are storing such longitude coordinate in our variable names dictionary with the key "Mesh2_node_x". So, we'd need to access the data variable as follows:
>>> x = grid.ds[grid.ds._var_names["Mesh2_node_x"]]

With the help of this function, we could also access it directly via Grid through a standardized name:
>>> x = grid.Mesh2_node_x